karma.conf.js

```diff
@@ -15,7 +15,7 @@ function makeDefaultConfig() {
     files: [KARMA_ENTRY],
     singleRun: !debug,
     autoWatch: debug,
-    frameworks: ['mocha', 'sinon-chai'],
+    frameworks: ['mocha', 'sinon-chai', 'webpack'],
```

Comment:
The [Karma docs] say this is a set of *test* frameworks - why have we added webpack?

[Karma docs]: http://karma-runner.github.io/6.3/config/configuration-file.html#frameworks

Reply:
my bad, I'll remove it